### ISSUE TITLE
Doesn't work with globally installed CLIs

### ISSUE THREAD

**Issue by sindresorhus** [vadimdemedes/import-jsx#19]
For example, when you install a CLI globally, the CLI binary ends up with a symlink in `/opt/homebrew/bin/emoj` that points to `/opt/homebrew/lib/node_modules/emoj/cli.js`. The problem is that `--loader=import-jsx` is resolved from `/opt/homebrew/bin/emoj` not `/opt/homebrew/lib/node_modules/emoj/cli.js`, so the `import-jsx` package cannot be found.

So `#!/usr/bin/env NODE_NO_WARNINGS=1 node --loader=import-jsx` doesn't actually work in practice unless you make a binary that is only used locally, which is rarely the case.

**Comment by vadimdemedes** [comment]
Yeah, that's an unfortunate and limiting tradeoff when using ESM loader hooks. My thinking was that this module might still be useful for writing React tests in AVA. I should remove the CLI section.

**Comment by sindresorhus** [comment]
> I should remove the CLI section.

And I would also mention that it won't work for globally installed CLIs, so nobody tries to be creative an re-invent was there in the readme before.
